com.logseq.Logseq.yml

```diff
@@ -1,12 +1,13 @@
+---
 id: com.logseq.Logseq
 runtime: org.freedesktop.Platform
-runtime-version: '22.08'
+runtime-version: "22.08"
 sdk: org.freedesktop.Sdk
 base: org.electronjs.Electron2.BaseApp
-base-version: '22.08'
+base-version: "22.08"
 sdk-extensions:
-  - org.freedesktop.Sdk.Extension.node16
-  - org.freedesktop.Sdk.Extension.openjdk8
+  - org.freedesktop.Sdk.Extension.node18
+  - org.freedesktop.Sdk.Extension.openjdk17
```

Comment:
In `logseq/.github/workflows/build.yml` JAVA_VERSION is 11

Reply:
@kanru it is. They have been ignoring updating deps until they automate the process. I have tested and used logseq with the LTS versions for Node and openjdk for more than 3 months. There is no issue in upgrading, and staying on outdated versions is not a good idea especially when there is no need to.